src/services/sponsorPppAdjusterService.ts

```
@@ -0,0 +1,117 @@
+  countryName: string;
+  basePrice: number;
+  pppIndex: number;
+  adjustedPrice: number;
+  adjustmentRatio: string; // e.g., "70% off" or "30% of base"
+  explanationText: string;
+}
+
+export class SponsorPppAdjusterService {
+  // PPP Index Table (USA = 1.0)
+  private pppIndexTable: Record<string, PppIndexRecord> = {
+    US: { countryCode: 'US', countryName: 'United States', pppIndex: 1.0, baseCurrency: 'USD', active: true },
```

ESLint

[error] Replace `·countryCode:·'US',·countryName:·'United·States',·pppIndex:·1.0,·baseCurrency:·'USD',·active:·true` with `⏎······countryCode:·"US",⏎······countryName:·"United·States",⏎······pppIndex:·1.0,⏎······baseCurrency:·"USD",⏎······active:·true,⏎···`

(prettier/prettier)